k8s/helmfile/env/staging/elasticsearch.values.yaml.gotmpl

```diff
@@ -10,6 +10,11 @@ imageTag: "6.8.23-wmde.6"
 
 esJavaOpts: "-Xms2g -Xmx2g"
 
+# this section can be removed once #1186 is merged
+esConfig:
+  elasticsearch.yml: |
+    cluster.info.update.timeout: 20s
```

Comment:
I think this would need to repeat all that there is in the production config as it is a plain string and won't get merged.